### PR TITLE
[Rec-IM] Fix Match Responsiveness

### PR DESCRIPTION
**Pull request by cpabbot**
## Match Page Responsiveness
Still not perfect, but much more clean and usable from before.

image
image

### CODE REVIEW

**Comment by silasw2** [comment]
New design: 
image
image

**Comment by amos-cha** [comment]
> New design: image image

Me being nit-picky, but out of the scope of this PR. 
(referring to the first image)
https://github.com/gordon-cs/RecIM-Docs/issues/179

File: src/views/RecIM/views/Match/index.jsx [hunk 1]
```diff
@@ -247,35 +297,12 @@ const Match = () => {
...
-              </Grid>
-            </GordonDialogBox>
             <GordonDialogBox
               title="Confirm Delete"
               open={openConfirmDelete}
               cancelButtonClicked={() => setOpenConfirmDelete(false)}
-              cancelButtonName="No, keep this team"
-              buttonName="Yes, delete this team"
+              cancelButtonName="No, keep this match"
+              buttonName="Yes, delete this match"
```
Comment (amos-cha) [inline src/views/RecIM/views/Match/index.jsx:305]: I wonder if there is a way to make the button a bit smaller, the text does seem long but this is personal opinion

**Review by amos-cha — APPROVED** [review]
Looks good. I made an issue for the small thing I noted.

**Pull request merged**